packages/client/src/generation/TSClient/common.ts

```diff
@@ -66,9 +66,9 @@ Prisma.validator = () => (val) => val
 /**
  * Shorthand utilities for JSON filtering
  */
-Prisma.DbNull = 'DbNull'
-Prisma.JsonNull = 'JsonNull'
-Prisma.AnyNull = 'AnyNull'
+Prisma.DbNull = Symbol.for('DbNull')
```

Comment:
Question: do we really need global symbols here or we could get away with local ones (`Symbol('DbNull')`)?